kotlinpoet/src/jvmMain/kotlin/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -326,14 +328,6 @@ public class FunSpec private constructor(
     @ExperimentalKotlinPoetApi
     override val contextReceiverTypes: MutableList<TypeName> = mutableListOf()
 
-    public fun addModifiers(vararg modifiers: KModifier): Builder = apply {
-      this.modifiers += modifiers
-    }
-
-    public fun addModifiers(modifiers: Iterable<KModifier>): Builder = apply {
-      this.modifiers += modifiers
-    }
```

Comment:
These need to be retained for binary compatibility, such as with `addAnnotation` below. And in other specs.